common/config/last9.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"errors"
+
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	l9OtlpEndpointKey   = "LAST9_OTLP_ENDPOINT"
+	l9OtlpAuthHeaderKey = "LAST9_OTLP_BASIC_AUTH_HEADER"
+)
+
+type MyDest struct{}
+
+func (m *MyDest) DestType() common.DestinationType {
+	// DestinationType defined in common/dests.go
+	return common.Last9DestinationType
+}
+
+func (m *MyDest) ModifyConfig(dest ExporterConfigurer, currentConfig *Config) error {
+	config := dest.GetConfig()
+	l9OtlpEndpoint, exists := config[l9OtlpEndpointKey]
+	if !exists {
+		return errors.New("Last9 OpenTelemetry Endpoint key(\"LAST9_OTLP_ENDPOINT\") not specified, Last9 will not be configured")
+	}
+
+	l9OtlpAuthHeader, exists := config[l9OtlpAuthHeaderKey]
+	if !exists {
+		return errors.New("Last9 OpenTelemetry Basic Auth Header key(\"LAST9_OTLP_BASIC_AUTH_HEADER\") not specified, Last9 will not be configured")
+	}
+
+	// to make sure that the exporter name is unique, we'll ask a ID from destination
+	exporterName := "otlp/last9-" + dest.GetID()
+	currentConfig.Exporters["otlp/last9"] = GenericMap{
+		"endpoint": l9OtlpEndpoint,
+		"headers": GenericMap{
+			"Authorization": l9OtlpAuthHeader,
+		},
+	}
```

Comment:
Since it is marked as a secret, I think this info will not show up in the config object.

It will be mapped to a k8s secret and will show up as environment variable
```suggestion
	// to make sure that the exporter name is unique, we'll ask a ID from destination
	exporterName := "otlp/last9-" + dest.GetID()
	currentConfig.Exporters["otlp/last9"] = GenericMap{
		"endpoint": l9OtlpEndpoint,
		"headers": GenericMap{
			"Authorization": "${LAST9_OTLP_BASIC_AUTH_HEADER}",
		},
	}
```